RestAPI_Automation/Tests/Tests.cs

```diff
@@ -0,0 +1,108 @@
+﻿using System.Diagnostics;
+using NUnit.Framework;
+using RestAPI_Automation.Requests;
+
+namespace RestAPI_Automation.Tests;
+
+public class Tests
+{
+    // Получить список юзеров, создать рандомный айдишник (нет в списке юзеров), отправить запрос с этим айдишником и проверить ответ – Not Found
+    [Test]
+    public void Task1()
+    {
+        var users = new APIHelper<CreatedUsers>();
+        // getting the list of users
+        var request1 = users.CreateGetRequest("/api/users?page=2");
+        var response1 = users.GetReponse(request1);
+        CreatedUsers content1 = users.GetContent<CreatedUsers>(response1);
+        // verifying that there are correct data in the response
+        Assert.AreEqual(2, content1.page);
+        Assert.AreEqual("Michael", content1.Data[0].first_name);
+
+        // creating random id and verifying that it is successfully created
+        string payload =
+            @"{
+                ""id"":""280"",
+                ""name"":""Mike""
+            }";
+        var user = new APIHelper<UserToCreate>();
+        var url = user.SetUrl("/api/users");
+        var request2 = user.CreatePostRequest(payload, "/api/users");
+        var response2 = user.GetReponse(request2);
+        UserToCreate content2 = user.GetContent<UserToCreate>(response2);
+        Assert.AreEqual("Mike", content2.name);
+        Assert.AreEqual(280, content2.id);
+
+        // sending request with new id
+        var createdUser = new APIHelper<Data>();
+        var request3 = createdUser.CreateGetRequest("/api/users?id=280");
+        var response3 = createdUser.GetReponse(request3);
+
+        // verifying that we got "Not Found" response
+        Assert.AreEqual("Not Found", response3.StatusDescription);
+    }
+
+    // Создать юзера и проверить ответ, в том числе время создания (между отправкой запроса и получением ответа)
+    [Test]
+    public void Task2()
+    {
+        string payload =
+            @"{
+                ""name"":""Mike"",
+                ""job"":""Team leader"",
+                ""id"":""222""
+            }";
```

Comment:
i think such data must be stored in model or dictionary
it can be used also below in assertions

Reply:
corrected